cmd/root/root.go

```diff
@@ -124,6 +126,14 @@ Stack Trace:
 %s`, version, r, string(trace))
 	}()
 
+	// Configure a telemetry logger and store it in the context.
+	ctx = telemetry.WithNewLogger(ctx)
+
+	// Detect if the CLI is running on DBR and store this on the context.
+	ctx = dbr.DetectRuntime(ctx)
```

Comment:
Why did this move?

If the telemetry function needs to access the outcome then I get it, but right now I don't think it does.

Reply:
I think you are right. We'll need it, though, after https://github.com/databricks/cli/pull/2438 once we start depending on `libs/dir` here.